steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,46 @@
+
+My manager was skeptical of the deadline - not because of my capability (because I'd done this exact thing in a previous role) but because of the people within the team.
+
+So began my journey of unofficial Quality Coach.
+
+The team consisted of a bunch of developers that had lots of experience of writing all the code and then throwing it over the fence, aka they only knew how to deliver changes within a Waterfall process. My challenge was to show them, educate them, and encourage them to take on board the philosophy of Agile and the Scrum methodology in particular. I should add there was a team manager but he was split across several teams and didn't have the time or experience of transforming a team to Agile.
+
+The second part of my approach focused on integrating Test Automation across the soon to be Agile/Scrum delivery lifecycle that the team would adopt. Experience had taught me a big lesson that not getting Test Automation embedded within the entire SDLC would stop all future progress to becoming an agile delivery team. Coupled with this there was the other elephant in the room 'testing is a shared team responsibility' that needed to be tackled quickly and effectively.
+
+The third part of my approach focused on the technologies and tooling for the team and I had assumed the developers would be excited at using shiny new tools, such as: Jenkins, SonarQube, Continuous Delivery, the AWS platform, and so on. I wasn't wrong in my assumption they couldn't get enough, they took all the training courses, all of them completed at least 2 levels of the AWS certification path. All that was missing was how they could create and make use of a Jenkins pipeline and that's where I could step in to help both with the design and initial implementation.
+
+Over the next 6 months I presented multiple workshops covering topics such as: Agile 101, What is Scrum anyway, delivering small pieces of value. Additionally I introduced techniques such as scenario example mapping and story pointing, and so on. I also did a lot of work on breaking down the "It's not my job to write the tests" mindset that was firmly embedded in the team, to help achieve this I introduced the team to the notion of working through the User story and creating Acceptance Criteria using the Gherkin format, we practiced this on a different basis. At no point during these exercies/sessions did I mention the words test coverage, cucumber, or even exploratory.
```

Comment:
How did the team find the workshops - what was their reaction?

Reply:
This was something I cut out of the original, due to length, I'll look to rework this into the story but not lose the emphasis.